app/src/containers/SearchPage/reducer.js

```diff
@@ -8,7 +10,31 @@ import {
 } from './action-types';
 import createReducer from '../../redux/create-reducer';
 
-const initialState = {
+export type Movie = {
```

Comment:
If you are sure you never mutate the object you can use `+` to make fields readonly. And also don't forget about exact typing with `|`.
```javascript
export type Movie = {|
  +id: number,
  +title: string,
  ...
  +genres: Array<string>,
|};
```